packages/local_auth/android/src/main/java/io/flutter/plugins/localauth/AuthenticationHelper.java

```diff
@@ -125,21 +137,27 @@ private void stop() {
   public void onAuthenticationError(int errorCode, CharSequence errString) {
     switch (errorCode) {
       case BiometricPrompt.ERROR_NO_DEVICE_CREDENTIAL:
-        completionHandler.onError(
-            "PasscodeNotSet",
-            "Phone not secured by PIN, pattern or password, or SIM is currently locked.");
-        break;
+        if (call.argument("useErrorDialogs")) {
+          showGoToSettingsDialog(
+              "Device credentials required",
```

Comment:
These  strings should come from Dart just like the rest of the status strings used below.

Reply:
Added to `AndroidAuthMessages` class